src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+const collum = document.querySelector('tbody').children;
+const tHead = document.querySelector('thead').children[0];
+const tFoot = document.querySelector('tfoot').children[0];
```

Comment:
try to use table methods instead
you can find table element with querySelector and then use tHead, tFoot. rows properties to find table elements

Reply:
https://learn.javascript.ru/dom-navigation#dom-navigation-tables